### PR TITLE
Add getOwnersForCollection endpoint + cleanup README

### PR DESCRIPTION
**Pull request by thebrianchen**
Adding `getOwnersForCollection()` endpoint along with some README cleanup.
- Note that developer return type is `owners` whereas the backend returns it under `ownerAddresses`. This is done to standardize with `getOwnersForNft()`, which also returns as `owners`.

### CODE REVIEW

File: src/api/nft-api.ts [hunk 1]
```diff
@@ -264,7 +266,7 @@ export async function getNftsForCollection(
 ): Promise<CollectionNftsResponse | CollectionBaseNftsResponse> {
   const withMetadata = omitMetadataToWithMetadata(options?.omitMetadata);
   const response = await requestHttpWithBackoff<
-    GetNftsForCollectionAlchemyParams,
+    GetOwnersForCollectionAlchemyParams,
```
Comment (alex-miao) [inline src/api/nft-api.ts:269]: Why is this within `getNftsForCollection`? Think you want a new function `getOwnersForCollection`.

File: src/api/nft-api.ts [hunk 2]
```diff
@@ -610,7 +661,7 @@ function omitMetadataToWithMetadata(
  *
  * @internal
  */
-interface GetNftsForCollectionAlchemyParams {
+interface GetOwnersForCollectionAlchemyParams {
   contractAddress: string;
   startToken?: string;
```
Comment (alex-miao) [inline src/api/nft-api.ts:666]: No need for `startToken` and `withMetadata` for `getOwnersForCollection`.

**Review by alex-miao — CHANGES_REQUESTED** [review]

File: src/api/nft-api.ts (hunk 1 above)
Comment (thebrianchen) [reply src/api/nft-api.ts]: oops, forgot to rename back. Thanks for catching!

**Review by alex-miao — COMMENTED** [review]
Code changes look good! Getting a CircleCI build error but will stamp once that gets fixed.

**Review by alex-miao — APPROVED** [review]
Approving since CircleCI fix is separate PR.

**Pull request merged**